test_runner/src/main/kotlin/ftl/util/Utils.kt

```diff
@@ -155,6 +155,10 @@ fun withGlobalExceptionHandling(block: () -> Int) {
                 System.err.println(t)
                 exitProcess(1)
             }
+            is FlankNoSuchFileException -> {
```

Comment:
This block is duplicated with L154, I think we should reuse code and and write multiple types for one branch.
IMO it would be even better to reuse logic for `FlankCommonException` (L135).

Reply:
Exception type changed to: FlankFatalError.